pkgs/development/python-modules/readme-renderer/default.nix

```
@@ -13,15 +13,15 @@
 
 buildPythonPackage rec {
   pname = "readme-renderer";
-  version = "43.0";
+  version = "44.0";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
```

```suggestion
  disabled = pythonOlder "3.9";
```
https://github.com/pypa/readme_renderer/blob/1d0497c37a6033d791c74e800590dcd0d34f6e08/pyproject.toml#L35